Tests/js/views/subitem/assets/ez-subitemgridview-tests.js

```diff
@@ -330,10 +332,63 @@ YUI.add('ez-subitemgridview-tests', function (Y) {
                     gridContainer.one('.contentType').getContent(),
                     "The grid item view template should receive the contentType"
                 );
-                
+
                 i++;
             }, this);
         },
+
+
+
+        "Should append the right number of grid items": function () {
+            var subitems = [
+                    this._getSubItemStruct(1),
+                    this._getSubItemStruct(2),
+                    this._getSubItemStruct(3),
+                    this._getSubItemStruct(4),
+                    this._getSubItemStruct(5),
+                    this._getSubItemStruct(6),
+                    this._getSubItemStruct(7),
+                    this._getSubItemStruct(8),
+                    this._getSubItemStruct(9),
+                    this._getSubItemStruct(10),
+                    this._getSubItemStruct(11),
+                    this._getSubItemStruct(12),
+                    this._getSubItemStruct(13),
+                    this._getSubItemStruct(14),
+
+                ],
+                container = this.view.get('container'),
+                gridItems,
+                i = 1,
+                offset = this.view.get('offset'),
+                button = this.view.get('container').one('.ez-subitemgrid-more');
+
+            this.view.set('subitems', subitems);
+            gridItems = container.all('.ez-subitemgrid-item');
+            Assert.areEqual(
+                this.view.get('limit'),
+                this.view.get('container').all('.ez-subitemgrid-content').size(),
+                "OoO"
+            );
+
+            button.set('disabled', false);
+            button.simulateGesture('tap', this.next(function () {
```

Comment:
using `tap` for this test complicates a bit your test. Why don't you change directly ̀the `subitems` attribute since the code you want to test is called in a `subitemsChange` event handler ?